proto/dydxprotocol/revshare/tx.proto

```diff
@@ -1,7 +1,31 @@
 syntax = "proto3";
 package dydxprotocol.revshare;
 
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "gogoproto/gogo.proto";
+import "dydxprotocol/revshare/params.proto";
+
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/revshare/types";
 
 // Msg defines the Msg service.
-service Msg {}
\ No newline at end of file
+service Msg {
+  // SetMarketMapperRevenueShare creates a new revenue share for a market
+  // mapper.
```

Comment:
nit: It doesn't create a new rev share right? Just overwrites the current rev share?